tck/src/main/java/org/reactivestreams/tck/support/PublisherVerificationRules.java

```diff
@@ -297,6 +297,18 @@
    */
   void optional_spec111_maySupportMultiSubscribe() throws Throwable;
   /**
+   * Ask for a single-element {@code Publisher} and subscribes to it twice.
+   * Each {@code Subscriber} requests for 1 element and checks if onNext or onComplete signals was received.
```

Comment:
Hm, you're right - sorry was a bit tired when reviewing that :) 
Indeed seems to cover all legal cases, and failure being the skip, so I think this covers everything.